recipes-mender/mender/files/mender.service

```diff
@@ -5,7 +5,7 @@ Description=Mender OTA update service
 Type=simple
 User=root
 Group=root
-ExecStart=/usr/bin/mender -daemon
+ExecStart=/usr/bin/mender -daemon -log-level=debug -log-file=/var/log/mender.log
```

Comment:
Why shouldn't debug be enabled for the time being?

I mean, the client is under heavily development, and it might be interesting to have it enabled by default.